README.md

```diff
@@ -42,43 +42,43 @@ Various relevant PRISM models (mostly conversions of the SBML files in `./SBML`)
 
 This circuit is part of the genetic circuits designed by the software tool Cello [1]. The circuit has three input arguments IPTG, aTc, and Ara and one output argument YFP.
 
-![Figure1](https://github.com/fluentverification/CaseStudies_StochasticModelChecking/blob/refactor/Figures/0x8E_Original.png "Figure 1")
+![Figure1](/Figures/0x8E_Original.png "Figure 1")
 
 In the laboratory, circuit 0x8E showed an unexpected, glitching behavior. Further analysis resulted in two more designs of the circuit as well as the analysis of its glitching behavior [2].
 
-![Figure2](https://github.com/fluentverification/CaseStudies_StochasticModelChecking/blob/refactor/Figures/0x8E_LogicHazardFree.png "Figure 2")
+![Figure2](/Figures/0x8E_LogicHazardFree.png "Figure 2")
 
-![Figure3](https://github.com/fluentverification/CaseStudies_StochasticModelChecking/blob/refactor/Figures/0x8E_TwoInverter.png "Figure 3")
+![Figure3](/Figures/0x8E_TwoInverter.png "Figure 3")
 
 There are two types of glitches. In one type, the output should remain in a low state during an input transition (Zero Glitch). In the other type, the output should remain at a high state (One Glitch) during an input transition. More information can be found in [1-2].
 
 ### Muller C-element
 
 This genetic circuit implements a state-holding gate called Muller C-element. If both inputs are high, its output goes high. If both inputs are low, the output is low. If the inputs are mixed, it retains its previous state. The three designs are the Majority, Speed_Independent, and Toggle Switch design [3].
 
-![Figure4](https://github.com/fluentverification/CaseStudies_StochasticModelChecking/blob/refactor/Figures/Majority.png "Majority Design")
+![Figure4](/Figures/Majority.png "Majority Design")
 
-![Figure5](https://github.com/fluentverification/CaseStudies_StochasticModelChecking/blob/refactor/Figures/Speed_Independent.png "Speed-Speed_Independent")
+![Figure5](/Figures/Speed_Independent.png "Speed-Speed_Independent")
```

Comment:
Change to `![Figure5](./Figures/Speed_Independent.png "Speed-Speed_Independent")`